src/foam/u2/crunch/CapabilityCardView.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.crunch',
+  name: 'CapabilityCardView',
+  extends: 'foam.u2.View',
+
+  implements: [ 'foam.mlang.Expressions' ],
+
+  requires: [
+    'foam.u2.Element',
+    'foam.u2.crunch.Style',
+    'foam.nanos.crunch.UserCapabilityJunction',
+    'foam.nanos.crunch.CapabilityJunctionStatus',
+    'foam.u2.view.ReadOnlyEnumView',
+  ],
+
+  imports: [
+    'user',
+    'userCapabilityJunctionDAO',
+  ],
+
+  documentation: `
+      A single card in a list of capabilities.
+  `,
+
+  methods: [
+    function initE() {
+      this.SUPER();
+      var self = this;
+
+      // Methods of Style all return the first argument for chaining
+      self.s = self.Style.create();
```

Comment:
copying to a single character psedo-instance property doesn't seem like a good practice. Why not just store in local variable called 's' if typing self.style is too long for you.

Reply:
I used this for the chaining below but couldn't name it `style` because Element already has that property. Although I've realized now that the chaining was kind of silly so I'll remove that and make it a local variable.